scripts/make_phyla_plots_AGP.py

```diff
@@ -87,14 +87,14 @@ def main(otu_table, mapping_data, cat_tables, output_dir, sample_type='fecal',
 
     # Names categories being plotted
     if sample_type == 'fecal':
-        michael_pollan = '000007108.1075657'
+        michael_pollan = '10317.000007108'
         cat_list = ['You', 'Average', 'Similar Diet', ' Similar BMI',
                     'Same Gender', 'Similar Age', 'Michael Pollan']
         order = ['Sample', 'Average', 'DIET_TYPE', 'BMI_CAT', 'SEX',
                  'AGE_CAT', 'MP']
 
     elif sample_type == 'skin':
-        michael_pollan = '000007113.1075702'
+        michael_pollan = '10317.000007113'
```

Comment:
This is only due to the inconsistent sample names in the actual submissions right, not due to a Pandas problem?